src/test/resources/logback-test.xml

```diff
@@ -6,7 +6,7 @@
         </encoder>
     </appender>
 
-    <root level="DEBUG">
+    <root level="WARN">
```

Comment:
I'm worried that we might miss something important when running tests in the CI.